src/features/permissions/logics/permissions.logics.ts

```diff
@@ -0,0 +1,20 @@
+import { PERMISSIONS_KEY } from "../constants/permissions.keys";
+import { PermissionEnum } from "../constants/permissions.scopes";
+
+export const getPermissions = () => {
+  const permissions = sessionStorage.getItem(PERMISSIONS_KEY);
```

Comment:
by making this pull from storage and not redux, theres no way to react to permissions changing in real-time. not required for this initial PR, but something to consider.

Reply:
true true, but then again, will permissions change in real-time 🤔  but I think we should still move it to redux to be consistent